.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Tag and Release
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
I think this should be triggered by a successful run of `Run tests, build image and push to GitHub Container Registry`.